### PR TITLE
Fix CI for latest nightly

### PR DESCRIPTION
**Pull request by rochala**

### CODE REVIEW

File: tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala [hunk 1]
```diff
@@ -952,7 +952,12 @@ class CompletionWorkspaceSuite extends BaseCompletionSuite {
     compat = Map(
       "2" -> """|fooBar: String
                 |fooBar - case_class_param.A: List[Int]
-                |""".stripMargin
+                |""".stripMargin,
+      ">=3.4.1-RC1-bin-20240201-hash-NIGHTLY" ->
+        """|fooBar: String
+           |fooBar: List[Int]
+           |fooBar(n: Int): A
```
Comment (tgodzik) [inline tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala:959]: Where is the last one coming from?

File: tests/cross/src/test/scala/tests/pc/CompletionSuite.scala [hunk 2]
```diff
@@ -368,6 +419,17 @@ class CompletionSuite extends BaseCompletionSuite {
...
+           |CertPathBuilderSpi - java.security.cert
+           |ProcessBuilderImpl - scala.sys.process
+           |CertPathBuilderResult - java.security.cert
+           |PKIXBuilderParameters - java.security.cert
+           |PooledConnectionBuilder - javax.sql
+           |CertPathBuilderException - java.security.cert
+           |PKIXCertPathBuilderResult - java.security.cert
+           |""".stripMargin,
+      ">=3.4.1-RC1-bin-20240201-hash-NIGHTLY" ->
+        """|ProcessBuilder - scala.sys.process
```
Comment (tgodzik) [inline tests/cross/src/test/scala/tests/pc/CompletionSuite.scala:432]: Why did ProcessBuilder disappear?

Comment (rochala) [reply tests/cross/src/test/scala/tests/pc/CompletionSuite.scala:432]: It is in scope, and compiler doesn't support fuzzy search. I have a fix for that that will ship next week.

File: tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala (hunk 1 above)
Comment (rochala) [reply tests/cross/src/test/scala/tests/pc/CompletionWorkspaceSuite.scala:959]: Completion for apply for fooBar in scope

**Review by tgodzik — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**